osu.Game/Overlays/Profile/Header/Components/GroupBadgeFlow.cs

```diff
@@ -25,7 +25,12 @@ public GroupBadgeFlow()
                 Clear(true);
 
                 if (user.NewValue?.Groups != null)
+                {
                     AddRange(user.NewValue.Groups.Select(g => new GroupBadge(g)));
+                    Show();
+                }
+                else
+                    Hide();
```

Comment:
This shouldn't really be here as the container will be 0x0 and invisible anyway
![image](https://user-images.githubusercontent.com/8269193/222115337-a6a0ae63-aa63-4c1f-8708-aada262bfaa5.png)